HomeWork1/main.swift

```diff
@@ -7,5 +7,90 @@
 
 import Foundation
 
-print("Hello, World!")
+// Задание 1: Напишите переменные и константы всех базовых типов данных: int, UInt, float, double, string. У чисел вывести их минимальные и максимальные значения.
+
+var temperature: Int = -10
+
+let floor: Int = 5
+
+var age: UInt = 10
+
+let growth: UInt = 165
+
+var cost1: Float = 100.5
```

Comment:
В нейминге в очень редких случаях используются цифры. Привыкай писать без них
```suggestion
var costOne: Float = 100.5
```